boards/xenvm/xenvm_xenvm_gicv3_defconfig

```diff
@@ -1,16 +0,0 @@
-# Enable UART driver
-CONFIG_SERIAL=y
```

Comment:
Think this file is referenced in documentation file

Reply:
You're right, thanks. Pushed.